apicast/src/management.lua

```diff
@@ -113,13 +113,28 @@ function _M.disabled()
   ngx.exit(ngx.HTTP_FORBIDDEN)
 end
 
+function _M.info()
```

Comment:
Yes it is separate commit. Not really useful without the post_action changes because there where no timers before.